datasets/brp_azure/inschrijvingen/v2.0.0.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "inschrijvingen",
+  "type": "table",
+  "version": "2.0.0",
+  "auth": "BRP/R",
+  "reasonsNonPublic": [
+    "5.1 1d: Bevat persoonsgegevens"
+  ],
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "identifier": "identificatie",
+    "required": [
+      "schema",
+      "identificatie"
+    ],
+    "display": "identificatie",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.3.0#/definitions/schema"
+      },
+      "identificatie": {
+        "type": "string",
+        "description": "identificatie, dit is of het Burgerservicenummer, Anummer of een zelfgemaakte identificatie door GOB"
+      },
+      "datumIngangBlokkeringPL": {
+        "type": "string",
+        "format": "date",
+        "description": "datum waarop de gemeente de PL naar een andere gemeente stuurt"
+      },
+      "datumOpschortingBijhouding": {
+        "type": "string",
+        "format": "date",
+        "description": "datum waarop de bijhouding van de PL is gestaakt"
+      },
+      "omschrijvingRedenOpschortingBijhouding": {
+        "type": "string",
+        "description": "aanduiding van de reden waarom de bijhouding van de PL is opgeschort"
+      },
+      "datumEersteInschrijvingGba": {
+        "type": "string",
+        "format": "date",
+        "description": "datum vanaf wanneer een persoon in de GBA is ingeschreven"
+      },
+      "gemeenteWaarPersoonskaartIs": {
+        "type": "object",
+        "properties": {
+          "code": {
+            "type": "string",
+            "description": "code van de gemeenteWaarPersoonskaartIs."
```

Comment:
Als dit veld betrekking heeft op kadastrale gemeente codes, zou je hier een relation kunnen maken met /brk:kadastralegemeentecodes 
Zie bijvoorbeeld het "aangeduidDoorKadastralegemeentecode" veld in brk/kadastraleobjecten
Dan zijn die 2 vanuit de api ook te relateren

Reply:
Ja, dat is de bedoeling

Reply:
For now als comment er bij gezet